.eslintrc.js

```diff
@@ -1,32 +1,40 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
-  parserOptions: {
-    ecmaVersion: 6,
-    ecmaFeatures: {
-      jsx: true,
-      experimentalObjectRestSpread: true,
-    },
-    sourceType: 'module',
-  },
-  extends: ['airbnb-base', 'plugin:react/recommended', 'prettier'],
-  plugins: ['react', 'jest'],
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    'plugin:jest/recommended',
+    'prettier',
+  ],
```

Comment:
moving more to recommended configs, I was able to get rid of most of our custom ones.